src/uu/cksum/Cargo.toml

```diff
@@ -1,17 +1,24 @@
 [package]
-name = "cksum"
+name = "uu_cksum"
 version = "0.0.1"
-authors = []
+authors = ["uutils developers"]
 license = "MIT"
+description = "cksum ~ (uutils) display CRC and size of input"
+
+homepage = "https://github.com/uutils/coreutils"
+repository = "https://github.com/uutils/coreutils/tree/master/src/uu/cksum"
+keywords = ["coreutils", "uutils", "cross-platform", "cli", "utility"]
+categories = ["command-line-utilities"]
+edition = "2018"
 
 [lib]
-name = "uu_cksum"
 path = "src/cksum.rs"
 
 [dependencies]
 libc = "0.2.42"
-uucore = "0.0.2"
+uucore = { version="0.0.4", package="uucore", git="https://github.com/uutils/uucore.git", branch="canary" }
```

Comment:
The `git` and `branch` attributes?
They are needed in order to test `uucore` changes without requiring prior publication of those changes to crates.io. Since `uucore` is under tested at the moment, I want to be able to test the utils against it before making permanent published changes.

Reply:
ok